doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -277,71 +311,97 @@ int main(int argc, char **argv)
   collision_objects.push_back(collision_object);
 
   // Now, let's add the collision object into the world
-  ROS_INFO("Add an object into the world");
+  ROS_INFO_NAMED("tutorial", "Add an object into the world");
   planning_scene_interface.addCollisionObjects(collision_objects);
 
-  /* Sleep so we have time to see the object in RViz */
-  sleep(2.0);
+  // Show text in Rviz of status
+  visual_tools.publishText(text_pose, "Add object", rvt::WHITE, rvt::XLARGE);
+  visual_tools.trigger();
 
-  // Planning with collision detection can be slow.  Lets set the planning time
-  // to be sure the planner has enough time to plan around the box.  10 seconds
-  // should be plenty.
-  move_group.setPlanningTime(10.0);
+  // Sleep so we have time to see the object in RViz
```

Comment:
Fixed.

Side note: is the collision object ActionServer capability you added available yet in the PlanningSceneInterface? Would be great to use that instead of these crappy sleep timers, but I saved that project for a future improvement (maybe you could add it?)